app/__init__.py

```diff
@@ -1,36 +1,23 @@
 from flask import Flask
-from flask_sqlalchemy import SQLAlchemy
-from flask_migrate import Migrate
+from .db import db, migrate
+from .models import customer, rental, video
+from .routes.customer_routes import customers_bp
+from .routes.rental_routes import rentals_bp
+from .routes.video_routes import videos_bp
 import os
-from dotenv import load_dotenv
 
-db = SQLAlchemy()
-migrate = Migrate()
-load_dotenv()
-
-def create_app(test_config=None):
+def create_app():
```

Comment:
```suggestion
def create_app(config=None):
```

Do we want to add this?

Reply:
Yes, it looks like I lost some of the work here to add and use the config, thanks for catching it! I'll push an update for this file after adding the suggestions on the PR.